eservices-file-generator/src/main/scala/it/pagopa/interop/eservicesfilegenerator/model/EService.scala

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.interop.eservicesfilegenerator.model
+
+import io.circe._
+import io.circe.generic.semiauto._
+import java.util.UUID
+
+final case class EService(
+  name: String,
+  eserviceId: UUID,
+  versionId: UUID,
+  technology: Technology,
+  state: State,
+  basePath: Seq[String],
+  producerName: String,
+  version: String
```

Comment:
Maybe it has been solved already, but why an `Int`? 
The `version` field is a string at the persistence level.

Reply:
The version is saved as String but de facto used as an Int (not sure why we did it this way, maybe for some kind of future-proofness?), and we are always converting it in the business logic (like finding the last version) or presentation (like sorting versions).
I didn't want to expose to external services an implementation that could be a technical debt.

But if you think the String type is part of a requirement, we can revisit this implementation (possibly in a future task, because we would need to make some changes to the probing services)

Reply:
Let's keep Int for the version.